File: Courses/1445/student_work/cody/countingwithfunctions.py
#!/usr/bin/env python
########################
#Counting with functions
#counts up or down using simple for loops with 
#adjustable parameters todecide where to count to and from
#includes a function count_from_thirty_to_ten() to make 
#people who want exactly what they asked for as opposed to an adjustable function 
#Cody Atkinson
#March 3, 2015
#
########################
def countup(lower,upper):
	for thing in range (lower, upper, 1):
		print(thing),
	
def countdown(upper,lower):	
	for thing in range (upper, lower,-1):
		print(thing),	

def count_from_thirty_to_ten():
	countdown(30,9)

count_from_thirty_to_ten()
countup(20,31)
print("\n")
countdown(30,9)

File: Courses/1445/student_work/cody/countingwithfunctions.c
#include <stdio.h>
/*
Counting with functons
counts with for loops placed into funtion
Cody Atkinson
March 3, 2015
*/
void count_to_20(void){        //beginning of definition
int i ;
for(i=20;i<=30;i++){
	printf("%d",i);

printf(" ");}
printf("\n");


}			                      	//end of definition

void count_from_30(void){
int i;
for(i=30;i>=10;i-=1){
	printf("%d",i);

printf(" ");}
printf("\n");

}

void main(void){                //beginning of main
count_to_20();
count_from_30();

}			                         //end of main
